backend/experiment/models.py

```diff
@@ -279,7 +273,77 @@ def max_score(self):
 
         return 0
 
+    def add_default_question_series(self):
+        """ Add default question_series to experiment"""
+        from experiment.rules import EXPERIMENT_RULES
+        for i,question_series in enumerate(EXPERIMENT_RULES[self.rules]().question_series):
+            qs = QuestionSeries.objects.create(
+                name = question_series['name'],
+                experiment = self,
+                index = i+1,
+                randomize = question_series['randomize'])
+            for i,question in enumerate(question_series['keys']):
+                qis = QuestionInSeries.objects.create(
+                    question_series = qs,
+                    question = Question.objects.get(pk=question),
+                    index=i+1)
+
 
 class Feedback(models.Model):
     text = models.TextField()
     experiment = models.ForeignKey(Experiment, on_delete=models.CASCADE)
+
+
+class Question(models.Model):
+
+    key = models.CharField(primary_key=True, max_length=128)
+    question = models.CharField(max_length=1024)
+    editable = models.BooleanField(default=True, editable=False)
+
+    def __str__(self):
+        return "("+self.key+") "+ self.question
+
+    class Meta:
+        ordering = ["key"]
+
+
+class QuestionGroup(models.Model):
+
+    key = models.CharField(primary_key=True, max_length=128)
+    questions = models.ManyToManyField(Question)
```

Comment:
I was hoping that people test it first, so the code would make things much clearer, because I think the interface is quite intuitive. `QuestionGroup` is just a collection of questions for convenience to easier add to the `QuestionSeries`, which belongs to `Experiment` and describes the question sequence in experiment.

Reply:
Docstrings would still be a good idea.